addons/website_sale_referral/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_referral_tracking_public,Referral Tracking Public Access,website_sale_referral.model_referral_tracking,base.group_public,1,0,0,0
```

Comment:
A public user is able to read a referral tracking, on which you have the token. The read privilege should be restricted to the group_system too.

Reply:
Done